python/CMakeLists.txt

```diff
@@ -61,6 +61,7 @@ set(_source_components
 
   MPACTPythonSources
   MPACTPythonExtensions
+  MPACTBenchmarkPythonSources
```

Comment:
hmmm, can we make it a separate library? This suppose to only include "core" infracture-related code.

Reply:
Sounds good. I added another module called mpact_benchmark in python package.